cuebot/src/main/java/com/imageworks/spcue/dao/postgres/DispatchQuery.java

```diff
@@ -88,7 +88,7 @@ public class DispatchQuery {
                     "LEFT JOIN (" +
                         "SELECT " +
                             "limit_record.pk_limit_record, " +
-                            "SUM(layer_stat.int_running_count) AS int_sum_running " +
+                            "SUM(CAST(layer_stat.int_running_count AS numeric)) AS int_sum_running " +
```

Comment:
Reading postgres docs raised some concerns regarding the performance of this conversion:

https://www.postgresql.org/docs/current/datatype-numeric.html :
> The type numeric can store numbers with a very large number of digits. It is especially recommended for storing monetary amounts and other quantities where exactness is required. Calculations with numeric values yield exact results where possible, e.g., addition, subtraction, multiplication. However, **calculations on numeric values are very slow compared to the integer types**, or to the floating-point types described in the next section.

Reply:
I think we need to dig a bit deeper to understand where this is coming from. Was the issue impacting every query or a specific subset?

Reply:
It seemed to impact every query that used a `SUM` on a `bigint` field. I didn't test all of these queries but once I saw it a few times I just went through and changed all of them, it seemed consistent.

[According to postgres docs](https://www.postgresql.org/docs/current/functions-aggregate.html) `SUM(bigint)` always returns a `numeric`, so I'm assuming something is going wrong in that conversion process somewhere.

It could be a problem specific to the embedded postgres, I don't think I was able to reproduce it anywhere else. And I can't get an older version of the embedded postgres on ARM so I wasn't able to try that.

Reply:
@DiegoTavares Any update here? Possible to do a test on your side to see if it produces any performance issues of note?

Reply:
Our dba did some tests and it looks like there's no performance impact. Besides that, we've submitted a ticket to EDB asking about the error. I'm okay in merging this in to unblock the release. If we find out an update to postgres can fix the issue later, we can revert this PR.

Reply:
@bcipriano I have few questions for you
1. Did you installed postgres 14.5 using following link https://www.enterprisedb.com/downloads/postgres-postgresql-downloads or using some other source ?
2. Did you got similar issue when you was on version 12.12 ? Are you able to test it on Postgresql12.12 on your new macbook ?

Reply:
@bcipriano Can you confirm the source of postgres download? can you try  https://www.enterprisedb.com/downloads/postgres-postgresql-downloads for download postgres ? Let me know how it goes with EDB 14.5?

Reply:
One thing to mention -- this only seems to happen in Docker. When I run tests directly on my ARM Macbook, they pass fine, in fact this PR isn't needed at all. It's only when I try to build the Cuebot Docker image (which does the Gradle build/test as part of the build process) that this problem appears.

> Did you installed postgres 14.5 using following link https://www.enterprisedb.com/downloads/postgres-postgresql-downloads or using some other source ?

Postgres is installed via Gradle/Maven. You can see the versions defined in `build.gradle`: https://github.com/AcademySoftwareFoundation/OpenCue/pull/1199/files#diff-7e57ab670111c27c23069909d0cc32c88d266c9a1f3e6ca38052c973418146fe

I don't know if it's possible use Postgres binaries from some other source with this embedded Postgres thing we're using.

> Did you got similar issue when you was on version 12.12 ? Are you able to test it on Postgresql12.12 on your new macbook ?

Same issue on 11.13.0 and 12.12.0. But only on Docker.

Reply:
Did a bit more research into this today:

* Tried the latest Postgres 15 binaries. Same issue.
* Tried a different base image (`almalinux` instead of `gradle`). Same issue.
* Created a Postgres container from the standard `postgres` image on my mac. Was not able to reproduce the issue.

Still kind of stumped. I filed https://github.com/zonkyio/embedded-postgres/issues/99 to see if those folks have any idea.